### DISCUSSION TITLE
Implement Wlroots protocols?

### DISCUSSION THREAD

**Discussion by meisme-dev** [CuarzoSoftware/Louvre#24] [Ideas]
Implementing Wlroots protocols like wlr-layer-shell and wlr-screencopy would allow many Wlroots-specific programs to be able to run on Louvre-based compositors. One of the biggest selling points of Wlroots is the application support. The Wlroots protocol xml files are available [here](https://gitlab.freedesktop.org/wlroots/wlroots/-/tree/master/protocol?ref_type=heads).

**Comment by ehopperdietzel** [comment]
Thank you for the suggestion. I plan to add those after finishing implementing the essentials. Regarding layershell, I need to carefully consider how to add it because it imposes a specific structure on compositors that not everyone may want. However, it will sure take some time before I implement it, and by then, I'll have a clearer idea. Thanks!
